bin/clean-jenkins-log

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cat build.log \
+    | sed \
+    -e 's|ha:////[a-zA-Z0-9/+=][a-zA-Z0-9/+=]*||' \
```

Comment:
This actually matches more than required (since `=` can only appear at the end of the string, not in the middle), but I think we're fine with it.